examples/mach_eval_examples/fluxlinkage_inductance_eval/fluxlinkage_inductance_evaluator.py

```diff
@@ -0,0 +1,103 @@
+import os
+import sys
+import numpy as np
+import matplotlib.pyplot as plt
+from time import time as clock_time
+
+os.chdir(os.path.dirname(__file__))
+sys.path.append("../../../")
+
+from mach_eval.analyzers.electromagnetic.flux_linkage_analyzer import FluxLinkageJMAG_Problem, FluxLinkageJMAG_Analyzer
+from mach_eval.analyzers.electromagnetic.inductance_analyzer import InductanceProblem, InductanceAnalyzer
+
+from mach_cad.tools import jmag as JMAG
+
+filepath = "C:/Users/dante/Documents/github_repos/eMach/examples/mach_eval_examples/fluxlinkage_inductance_eval"
```

Comment:
This hard coded path won't work for other people. Can you please use relative paths?